src/main/java/uk/gov/hmcts/reform/sscs/robotics/RoboticsJsonMapper.java

```diff
@@ -142,6 +176,12 @@ private static JSONObject buildHearingOptions(HearingOptions hearingOptions) {
             hearingArrangements.put("other", hearingOptions.getOther());
         }
 
+        buildExcludedDates(hearingOptions, hearingArrangements);
```

Comment:
Is this supposed to be here?
We already have it with the datesCantAttend with "hearingArrangements".
Is it meant to be added twice?

Reply:
Should the call be removed in line 136?

Reply:
How come? They are needed in 2 different parts of the json